lib/cli.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
```

Comment:
to gitignore

Reply:
Why the content of the `lib` folder is the application itself